docs/src/pages/docs/functions/combinators.md

```diff
@@ -64,6 +64,119 @@ safeYell(42)
 //=> Left 42
 ```
 
+#### compose2
+
+`crocks/combinators/compose2`
+
+```haskell
+compose2 :: (c -> d -> e) -> (a -> c) -> (b -> d) -> a -> b -> e
+```
+
+`compose2` allows for composition between a `binary` function and
+two `unary` functions. `compose2` takes a `binary` function followed by
+two `unary` functions and returns a `binary` function that maps the first
+argument with the first `unary` and the second with the second, passing
+the results to the given `binary` and returning the result.
```

Comment:
Really not married to this description, it feels a little flat, lmk if anyone can make it better

Reply:
I think this is good as is, gets to the point and provides the info someone would need to wrap their head around it.